Sources/Implementation/DefaultNotificationCenter.swift

```diff
@@ -17,33 +17,60 @@
 import Foundation
 
 public class DefaultNotificationCenter: OPTNotificationCenter {
-    public var notificationId: Int = 1
-    var notificationListeners = [Int: (Int, GenericListener)]()
+    public var notificationId: Int {
+        get {
+            return notifications.notificationId
```

Comment:
We need sync-er for allowing access to notificationId (public), which can cause thread conflict with incrementNotificationId.

Reply:
i would prefer to remove this access all together.  it doesn't make sense.

Reply:
Yep, this should be removed, but it was open to public. Lets keep it for now and clean it up on the next major release.